packages/client/src/views/VerifyCertificate/useVerificationRecordDetails.ts

```diff
@@ -0,0 +1,126 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
+ */
+
+import { gql, useQuery } from '@apollo/client'
+import { Query } from '@client/utils/gateway'
+import { useParams } from 'react-router'
+
+const FETCH_RECORD_DETAILS_FOR_VERIFICATION = gql`
+  query fetchRecordDetailsForVerification($id: String!) {
+    fetchRecordDetailsForVerification(id: $id) {
+      ... on BirthRegistration {
+        id
+        child {
+          name {
+            firstNames
+            familyName
```

Comment:
Hm I must have missed adding middle name field here. Will need to open a PR on the release branch. We really need to refactor the queries to use fragments otherwise these types of bugs will keep coming